crud/src/main/java/com/redhat/lightblue/mediator/Mediator.java

```diff
@@ -115,7 +116,13 @@ public Mediator(Metadata md,
      * implementation can perform further validations.
      */
     @StopWatch(loggerName = "stopwatch.com.redhat.lightblue.mediator.Mediator", sizeCalculatorClass = "com.redhat.lightblue.mediator.ResponsePayloadSizeCalculator")
-    public Response insert(InsertionRequest req) {
+    public Response insert(InsertionRequest req, RequestMetrics metrics) {
```

Comment:
Instead of passing RequestMetrics to each method, can we just pass it in the constructor?